Solidus-Swift-SDK/EndpointsBase.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  EndpointsBase.swift
+//  Solidus-Swift-SDK
+//
+//  Created by Francisco Flores on 5/3/16.
+//  Copyright © 2016 Omar Guzman. All rights reserved.
+//
+
+import Foundation
+
+struct EndpointsBase {
+    static var products = "/api/products"
```

Comment:
If the _solidus_ engine is mounted in that _url_ the request is gonna be like _www.store.com/spree/api/products_